main.yml

```diff
@@ -161,6 +161,20 @@ landing:
   channel: '#flip-mode-squad'
   deployment_repo: https://github.com/RedHatInsights/landing-page-frontend-build
 
+migrations:
+  title: Migrations
+  api: 
+    versions:
+      - v1
+  frontend: 
+    title: Migrations
+    sub_apps:
+      - id: migration-analytics
+        title: Migration Analytics
+        group: migrations
```

Comment:
I may be mistaken, but I don't think you need the `group` tag for this one.

Reply:
removed